tree/treeplayer/inc/TTreeReaderValue.h

```diff
@@ -132,14 +132,22 @@ class TTreeReaderValue: public ROOT::Internal::TTreeReaderValueBase {
       TTreeReaderValueBase(&tr, branchname,
                            TDictionary::GetDictionary(typeid(NonConstT_t))) {}
 
+   /// Return a pointer to the value of the current entry.
+   /// Return a nullptr and print an error if no entry has been loaded yet.
+   /// The returned address is guaranteed to stay constant while a given TTree is being read from a given file.
+   /// It might change when the underlying TTree/TFile is switched, e.g. when a TChain switches files.
    T* Get() {
       if (!fProxy){
          Error("TTreeReaderValue::Get()", "Value reader not properly initialized, did you remember to call TTreeReader.Set(Next)Entry()?");
-         return 0;
+         return nullptr;
       }
       void *address = GetAddress(); // Needed to figure out if it's a pointer
       return fProxy->IsaPointer() ? *(T**)address : (T*)address; }
+   /// Return a pointer to the value of the current entry.
+   /// Equivalent to Get.
    T* operator->() { return Get(); }
+   /// Return a reference to the value of the current entry.
+   /// Equivalent to dereferencing the pointer returned by Get. Behavior is undefined if no entry has been loaded yet.
```

Comment:
"Behavior is undefined" is the correct technical term - "will likely crash" is the more user-oriented one :-) I don't have a strong preference here... in principle this should throw if `Get()` returns `nullptr`.

Reply:
will add a note about the crashing :)